### DISCUSSION TITLE
abu-beziehungen-und-zusammenleben

### DISCUSSION THREAD

**Discussion by mxrni** [mxrni/marni-markdown#1] [General]
comment section for abu-beziehungen-und-zusammenleben

**Comment by jubr001** [comment]
lutsch eier

**Reply by 69lutscher69** [reply]
womp womp

**Reply by jubr001** [reply]
22:00 Schaa Bahof Moruk

**Reply by mxrni** [reply]
"Ich kann das alles nicht mehr" ~ Zitat Eros

**Comment by jubr001** [comment]
Jan bitte korrigiera:

    "ausseer" -> "ausser"
    "Volljährig" -> "volljährig"
    "zustimmung" -> "Zustimmung"
    "änderung" -> "Änderung"

Danke gell

**Reply by janvetsch** [reply]
Mach selber jubr
